### PR TITLE
Create My Session List Page

### PR DESCRIPTION
**Pull request by kwmt**
## Issue
- none

## Overview (Required)

**I created my session list page, but if you don't need, please close this pull request.**

- I think I want to see my session list page. Because when I saved many sessions, I didn't know what session I saved.
- In this page, I want to check WHEN, WHERE, WHO and WHAT.

If you will merge, I couldn't decide about some points below:

- my session list layout design.
- the way to display this page (now  from menu of session page)
- description text if the list is empty

## Links
- 

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by konifar** [comment]
:eyes:

**Comment by konifar** [comment]
Thanks for good proposal! Interesting. Basically I agree your idea.

> my session list layout design.

As you mentioned, it's better to create other item layout and use it.
But it's just improvement task. For this PR, no need to fix.

> In this page, I want to check WHEN, WHERE, WHO and WHAT.

Sorry, I can't understand 🙇 Can you explain more?

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -91,6 +91,14 @@
                 android:theme="@style/AppTheme.NoActionBar.Splash"
                 />
 
+        <activity
+                android:name=".view.activity.MySessionsActivity"
+                android:configChanges="keyboard|keyboardHidden|screenLayout|screenSize|orientation"
+                android:exported="false"
+                android:label="Mysessions"
```
Comment (konifar) [inline app/src/main/AndroidManifest.xml]: Please extract this to `strings.xml`

File: app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/MySessionsFragment.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+public class MySessionsFragment extends BaseFragment {
+
+    public static final String TAG = MySessionsFragment.class.getSimpleName();
+
+    @Inject
+    MySessionsViewModel viewModel;
+
+    @Inject
+    CompositeDisposable compositeDisposable;
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/MySessionsFragment.java]: Actually, it's better to move this `compositeDisposable` to `MySessionsViewModel`.
If you want to refactor, `ContributorsFragment` and `ContributorsViewModel` help you.

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/MySessionsFragment.java]: Ah, sorry. It's my misunderstanding 🙇 
Maybe this `compositeDisposable` is unnecessary here.

File: app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/MySessionsViewModel.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+import io.reactivex.disposables.Disposable;
+import timber.log.Timber;
+
+public final class MySessionsViewModel extends BaseObservable implements ViewModel {
+
+    private static final String TAG = MySessionsViewModel.class.getSimpleName();
+
+    private MySessionsRepository mySessionsRepository;
+
+    private final ObservableList<MySessionViewModel> mySessionViewModels;
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/MySessionsViewModel.java]: 👍

**Comment by kwmt** [comment]
>> In this page, I want to check WHEN, WHERE, WHO and WHAT.
> Sorry, I can't understand 🙇 Can you explain more?

Sorry, in japanese 😓 
MySessionページで僕が見たいのは、いつ・どこで・だれが・なにをセッションするのかを一覧でみれたいいかなと思って今はそのようにしていますが、不要なものがあったり、必要なものがあるかもしれません。。。

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (kwmt) [inline app/src/main/AndroidManifest.xml]: Ah! I fixed it!

File: app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/MySessionsFragment.java (hunk 2 above)
Comment (kwmt) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/MySessionsFragment.java]: Oh... I fixed it!

**Comment by konifar** [comment]
@kwmt Understood! Thanks!
説明ありがとうございます！了解しました 👍

File: app/src/main/res/layout/view_my_session.xml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+    <data>
+
+        <variable
+                name="viewModel"
+                type="io.github.droidkaigi.confsched2017.viewmodel.MySessionViewModel"
+                />
+    </data>
+
+    <android.support.constraint.ConstraintLayout
```
Comment (konifar) [inline app/src/main/res/layout/view_my_session.xml]: Currently, this layout is fine 😉

File: app/src/main/res/layout/fragment_my_sessions.xml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+                android:layout_height="wrap_content"
+                android:background="@color/white"
+                android:visibility="@{viewModel.recyclerViewVisibility}"
+                />
+
+        <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:padding="@dimen/space_16dp"
+                android:text="@string/my_session_empty_description"
```
Comment (konifar) [inline app/src/main/res/layout/fragment_my_sessions.xml]: Can you add these attributes?

```xml
android:textApperance="@style/TextSubheading"
android:textColor="@color/grey500"
android:layout_centerInParent="true"
```

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -39,6 +39,8 @@
     <string name="session_unchecked">Unchecked this session</string>
     <string name="session_check">Check</string>
     <string name="session_uncheck">Uncheck</string>
+    <string name="my_session">My Session</string>
+    <string name="my_session_empty_description">You don\'t save a my session. To see my session, please you save it in session detail page.</string>
```
Comment (konifar) [inline app/src/main/res/values/strings.xml]: Can you change this message to this one? 🙇 
https://github.com/konifar/droidkaigi2016/blob/master/app/src/main/res/values/strings.xml#L110

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -39,6 +39,8 @@
     <string name="session_unchecked">Unchecked this session</string>
     <string name="session_check">Check</string>
     <string name="session_uncheck">Uncheck</string>
+    <string name="my_session">My Session</string>
```
Comment (konifar) [inline app/src/main/res/values/strings.xml]: Can you separate the section like this?

```xml
<string name="session_uncheck">Uncheck</string>

<!-- Sessions -->
<string name="my_session">My Session</string>
```

File: app/src/main/res/layout/fragment_my_sessions.xml (hunk 5 above)
Comment (kwmt) [inline app/src/main/res/layout/fragment_my_sessions.xml]: I added these.

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (kwmt) [inline app/src/main/res/values/strings.xml]: Thanks! I changed to it.

File: app/src/main/res/values/strings.xml (hunk 7 above)
Comment (kwmt) [inline app/src/main/res/values/strings.xml]: I fixed it.

File: app/src/main/res/layout/fragment_my_sessions.xml (hunk 5 above)
Comment (konifar) [inline app/src/main/res/layout/fragment_my_sessions.xml]: Perfect 💯 Thanks!

**Comment by konifar** [comment]
LGTM! 👍 👍  Thanks for contribution and quick fix!!

**Comment by konifar** [comment]
Hope CI success 🙏

**Comment by kwmt** [comment]
Thanks for quick review! 🙇

**Pull request merged**